assets/xml/objects/object_boss01.xml

```diff
@@ -1,189 +1,232 @@
-        <Limb Name="object_boss01_Standardlimb_0221F0" Type="Standard" EnumName="OBJECT_BOSS01_2_LIMB_09" Offset="0x221F0" />
-        <Limb Name="object_boss01_Standardlimb_0221FC" Type="Standard" EnumName="OBJECT_BOSS01_2_LIMB_0A" Offset="0x221FC" />
-        <Limb Name="object_boss01_Standardlimb_022208" Type="Standard" EnumName="OBJECT_BOSS01_2_LIMB_0B" Offset="0x22208" />
-        <Limb Name="object_boss01_Standardlimb_022214" Type="Standard" EnumName="OBJECT_BOSS01_2_LIMB_0C" Offset="0x22214" />
-        <Limb Name="object_boss01_Standardlimb_022220" Type="Standard" EnumName="OBJECT_BOSS01_2_LIMB_0D" Offset="0x22220" />
-        <Limb Name="object_boss01_Standardlimb_02222C" Type="Standard" EnumName="OBJECT_BOSS01_2_LIMB_0E" Offset="0x2222C" />
-        <Limb Name="object_boss01_Standardlimb_022238" Type="Standard" EnumName="OBJECT_BOSS01_2_LIMB_0F" Offset="0x22238" />
-        <Limb Name="object_boss01_Standardlimb_022244" Type="Standard" EnumName="OBJECT_BOSS01_2_LIMB_10" Offset="0x22244" />
-        <Limb Name="object_boss01_Standardlimb_022250" Type="Standard" EnumName="OBJECT_BOSS01_2_LIMB_11" Offset="0x22250" />
-        <Limb Name="object_boss01_Standardlimb_02225C" Type="Standard" EnumName="OBJECT_BOSS01_2_LIMB_12" Offset="0x2225C" />
-        <Limb Name="object_boss01_Standardlimb_022268" Type="Standard" EnumName="OBJECT_BOSS01_2_LIMB_13" Offset="0x22268" />
-        <Limb Name="object_boss01_Standardlimb_022274" Type="Standard" EnumName="OBJECT_BOSS01_2_LIMB_14" Offset="0x22274" />
-        <Skeleton Name="object_boss01_Skel_0222D0" Type="Flex" LimbType="Standard" LimbNone="OBJECT_BOSS01_2_LIMB_NONE" LimbMax="OBJECT_BOSS01_2_LIMB_MAX" EnumName="ObjectBoss012Limb" Offset="0x222D0" />
-        <Animation Name="object_boss01_Anim_022550" Offset="0x22550" />
+        <!-- Odolwa Animations -->
+        <Animation Name="gOdolwaVerticalSlashAnim" Offset="0xC44" /> <!-- Original name is "jmp_at01" -->
+        <Animation Name="gOdolwaHorizontalSlashAnim" Offset="0x1884" /> <!-- Original name is "jmp_at02" -->
+        <Animation Name="gOdolwaAtAttentionAnim" Offset="0x1A14" /> <!-- Unused. Original name is "jmp_base" -->
+
+        <!-- Odolwa Limb DisplayLists -->
+        <DList Name="gOdolwaPelvisDL" Offset="0x5E80" />
+        <DList Name="gOdolwaHeadDL" Offset="0x5F98" />
+        <DList Name="gOdolwaCenterHairTipDL" Offset="0x61E8" />
+        <DList Name="gOdolwaCenterHairMiddleDL" Offset="0x62A8" />
+        <DList Name="gOdolwaCenterHairBaseDL" Offset="0x6368" />
+        <DList Name="gOdolwaLeftHairTipDL" Offset="0x6408" />
+        <DList Name="gOdolwaLeftHairMiddleDL" Offset="0x64C8" />
+        <DList Name="gOdolwaLeftHairBaseDL" Offset="0x6588" />
+        <DList Name="gOdolwaRightHairTipDL" Offset="0x6628" />
+        <DList Name="gOdolwaRightHairMiddleDL" Offset="0x66E8" />
+        <DList Name="gOdolwaRightHairBaseDL" Offset="0x67A8" />
```

Comment:
I don't think this is meant to be hair, I assume it's some plumes attached to the mask or something.

Reply:
We've been calling it hair in other objects (like `bsmask`), but yeah, it's not really hair. I'll try to change it everywhere.